website_sale_qty/static/src/js/website_sale_qty.js

```diff
@@ -0,0 +1,47 @@
+/**
+*    Copyright 2016 LasLabs Inc.
+*    License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+**/
+
+odoo.define('website_sale_qty', function(require) {
+    'use strict';
+
+    var base = require('web_editor.base');
+    base.ready().done(function() {
+        var radios = $('.js-pqt-input');
+
+        if (radios.length > 0) {
+            var radio_map = {};
+            radios.each(function(index, element) {
+                var $element = $(element);
+                radio_map[$element.val()] = $element;
+            });
+            var quantity_input = $('.css_quantity input');
+            var quantity_toggles = $('.css_quantity a');
+
+            // If present, this input should be selected by default
+            if (1 in radio_map) {
+                radio_map[1].prop('checked', true);
+            }
+
+            radios.click(function(event) {
+                quantity_input.val($(event.target).val());
```

Comment:
Good catch. At least in Chrome, `event.target` works just fine here because of the way that a click on a `<label>` element produces a click on the associated `<input>`, but `event.currentTarget` would be more universal.